apps/src/studio/projectile.js

```diff
@@ -213,3 +212,14 @@ Projectile.prototype.moveToNextPosition = function () {
   this.x = next.x;
   this.y = next.y;
 };
+
+/**
+ * Change visible opacity of this projectile.
+ * @param {number} newOpacity (between 0 and 1)
+ * @override
+ */
+Projectile.prototype.setOpacity = function (newOpacity) {
```

Comment:
It is in Collidable, but it's overridden here in Projectile (and similarly in Item) because they create and reference their image element(s) differently.